lib/stat_tracker.rb

```diff
@@ -12,11 +12,11 @@ def self.from_csv(locations)
     @stat_tracker = StatTracker.new(@data_array)
   end
 
-  def initialize(data_array)
+  def initialize(data_array, games = nil, teams = nil, game_teams = nil)
     @data_array = data_array
-    @games = CSV.read @data_array[0], headers: true, header_converters: :symbol
```

Comment:
Not having parenthesis during this method call caused a bug when I added the `||` clause